tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1699,7 +1719,7 @@ Status IrEmitterUnnested::EmitLoopFusion(mlir::Operation* op) {
       if (auto broadcast = mlir::dyn_cast<mlir::mhlo::BroadcastInDimOp>(op)) {
         if (broadcast.broadcast_dimensions().empty() ||
             // More then 2 bit inputs cause one speed regression.
-            (row_vectorized && num_big_inputs <= 2)) {
+            (row_vectorized && num_big_inputs <= 3)) {
```

Comment:
At least we need to update the comment. Could you extract this into a constant? Was it tuned manually? On which examples?

Reply:
I can remove this now. I added that late to remove a small V100 slowdown.
It was needed for the fusion_5_simpler* HLO in the description.
Do you want me to remove it or keep it?